.gitignore

```diff
@@ -24,22 +22,20 @@ out/
 
 .editorconfig
 
-# syncthing
-.stignore
```

Comment:
@markrmiller  added this one once

Reply:
I see no harm in this, as it reduces the chance of any future contributor accidentally committing this file if they use such tool. A few extra lines in gitignore is tolerable. We’re not adding every possible tool in here, but devs will add such as they experience the pain with a certain tool.